handlers/strike.py

```diff
@@ -0,0 +1,43 @@
+import os
+from jsonpath_rw import jsonpath, parse
+
+class Strike():
+	def handleCommand(self, command):
+	
+		if command.command == "getStrikes"
+			return getStrikesString(self)
+		userId = parse('$.message.from.id').find(command.parent)
+		strikeUser = command.args
+		if not userId or not strikeUser:
```

Comment:
please validate the user is in the channel.
also if not found return that string.

Reply:
Yeah thats next, a lot harder to do :( pending